ajax/downVoteAjax.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Quick note on the PR: I'd switch it to point at staging instead of master. You'll almost always want to merge all your feature branches into staging, and only merge staging into master.

Reply:
Hi Tyler! So staging has my blog files, which are entirely different from this events project. Would I still point it to staging? I was thinking after this PR, I could make "started-6/14" act as a staging branch, and then work everything off this branch.